src/components/seo.js

```diff
@@ -25,32 +25,40 @@ function SEO({ description, lang, meta, keywords, title }) {
       meta={[
         {
           name: `description`,
-          content: metaDescription,
+          content: 'A place for Q&A and free mentorship',
+        },
+        {
+          property: `og:title`,
+          content: 'Mentor Night | Free Code Camp',
+        },
+        {
+          property: `og:description`,
+          content: 'A place to be coached or to coach for free along your developer journey',
+        },
+        {
+          property: `og:type`,
+          content: `website`,
+        },
+        {
+          name: `twitter:card`,
+          content: `summary`,
+        },
+        {
+          name: `twitter:creator`,
+          content: '@freeCodeCamp,'
+        },
+        {
+          name:'url',
+          content:"https://nashvillefcc.github.io/mentor-night-site/", //Added line 52 and it's key:value pair because the content key was present in other examples
+        },
+        {
+          name: `twitter:title`,
+          content: 'Mentor Night on Twitter',
```

Comment:
Like the `metaDescription`, it might be better to derive the title from `gatsby-config.js` using `site.siteMetadata.title`.

Reply:
Actually, is `twitter:title` a supported tag? I didn't see it on the Twitter specifications site:
https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started

But that doesn't seem like the most comprehensive documentation. Is there more?